.github/workflows/ci-cpp-build.yaml

```diff
@@ -54,6 +53,12 @@ jobs:
         id: check_changes
         run: |
           echo "::set-output name=docs_only::${{ fromJSON(steps.changes.outputs.all_count) == fromJSON(steps.changes.outputs.docs_count) && fromJSON(steps.changes.outputs.docs_count) > 0 }}"
+          echo "::set-output name=cpp_only::${{ fromJSON(steps.changes.outputs.all_count) == fromJSON(steps.changes.outputs.cpp_count) && fromJSON(steps.changes.outputs.cpp_count) > 0 }}"
```

Comment:
```suggestion
```

This is exact the CPP workflow. We can remove the `cpp_only` output instead.

Reply:
sure, it was missing from the go workflow so it was added just to be consistent.